src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +15,41 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
```

Comment:
Could be simplified with single line.
smth like this
override fun greet() = ENGLISH_GENERAL_GREETING